### PR TITLE
delete buddy_allocator_test_data to make repo clean

### PR DESCRIPTION
**Pull request by tianshuo78520a**
### PR types

Function optimization
### PR changes

Others 
### Describe

delete testdata

### CODE REVIEW

File: paddle/fluid/memory/detail/CMakeLists.txt [hunk 1]
```diff
@@ -12,9 +20,25 @@ cc_library(buddy_allocator SRCS buddy_allocator.cc DEPS memory_block system_allo
 
 cc_test(buddy_allocator_test SRCS buddy_allocator_test.cc DEPS buddy_allocator)
 
+function(download_and_uncompress INSTALL_DIR URL FILENAME)
+  message(STATUS "Download test stuff from ${URL}/${FILENAME}")
+  ExternalProject_Add(
+      PREFIX                ${INSTALL_DIR}
```
Comment (zhwesky2010) [inline paddle/fluid/memory/detail/CMakeLists.txt]: ExternalProject_Add第一个字符是target的name，可以随便取一个，比如buddy_allocator_test_data，第二个参数是${EXTERNAL_PROJECT_LOG_ARGS}。这里把PREFIX当做了name

Comment (tianshuo78520a) [reply paddle/fluid/memory/detail/CMakeLists.txt]: 好的已经修改了

**Review by zhhsplendid — APPROVED** [review]
LGTM on code, but why is the PR type 'breaking change'?

**Comment by tianshuo78520a** [comment]
> LGTM on code, but why is the PR type 'breaking change'?

Function optimization has been modified

File: paddle/fluid/memory/detail/CMakeLists.txt [hunk 2]
```diff
@@ -12,9 +14,27 @@ cc_library(buddy_allocator SRCS buddy_allocator.cc DEPS memory_block system_allo
 
 cc_test(buddy_allocator_test SRCS buddy_allocator_test.cc DEPS buddy_allocator)
 
+FUNCTION(file_download_and_uncompress URL NAME)
+  MESSAGE(STATUS "Download dependence[${NAME}] from ${URL}")
```
Comment (luotao1) [inline paddle/fluid/memory/detail/CMakeLists.txt:18]: 这句打印可以去掉

**Pull request merged**